pymedusa/Medusa
# [Python3] Changes in python2 code to make it python3 compatible (part1)

This PR only prepares the medusa source code to be compatible with python3. The most important change from python2 to python3 is the string/unicode/bytes support. Code still non working in python3.
**This PR is the 1st in [Python3] series.**
Main discussion: #4975

**You wanna help?**
* Test with python2. Everything should work as usual. Especially test strings/series with non ascii characters.

**Instructions to test**
* `git clone https://github.com/ngosang/Medusa.git`
* `cd Medusa`
* `git pull` and `git checkout feature/py3_3`
* edit `config.ini` and change `debug = 1`
* `python -V` or `python2 -V`
* `python start.py`  or `python2 start.py`
* Try to enable/configure all features in the webui

Disclaimer:
> I'm testing careful with both python2 and python3 interpreters. With python2 you should expect the same functionally than development branch. Python3 is a long term goal, first versions won't be functional or even won't run.
